ui/components/multichain/account-list-item/account-list-item.js

```diff
@@ -127,10 +126,20 @@ export const AccountListItem = ({
                   position="bottom"
                   wrapperClassName="multichain-account-list-item__tooltip"
                 >
-                  {identity.name}
+                  <ButtonLink
+                    onClick={onClick}
+                    className="multichain-account-list-item__account-name"
+                  >
+                    {identity.name}
+                  </ButtonLink>
```

Comment:
I think `<Text as="button">` we want to keep the Text styling but change the underlying element. I think you will still need to add some custom styles here via Box props or a className for the hover states though

Reply:
That isn't working as it bricks the `ellipsis` functionality.  Also, using `ButtonLink` requires only one CSS change, while using `as="button"` requires some background and padding work.